mamba_gator/envmanager.py

```diff
@@ -336,6 +336,29 @@ async def create_env(self, env: str, *args) -> Dict[str, str]:
             return {"error": output}
         return output
 
+    async def create_explicit_env(self, name: str, explicit_list: str) -> Dict[str, str]:
```

Comment:
This is a duplication of `import_env` line 385. Could you merge the two as the use of `flush` is definitely better than the implementation in `import_env`?

Reply:
I had tried that, but discovered the command needs to be slightly different: the explicit version needs the `env` argument removed.  We could still reuse `import_env`, but we would have to add an argument to it.

Reply:
Or we could inspect the file content and look for the line `@EXPLICIT` to determine if it's an explicit list.

Reply:
Sorry I miss the `env` part in the command. And without testing it I miss the fact that the final format was different of the one edited by the user.
Thanks for updating the code to look for `@EXPLICIT` and still use import.